requirements.txt

```diff
@@ -1,6 +1,6 @@
 h5py>=2.8
 matplotlib>=1.4.3
-numpy==1.15.4
+numpy
```

Comment:
Do we want to set a lower-bound?

Reply:
Would make sense as we had them pinned. I think numpy>=1.15.4 and tables >=3.5.1